docs/upgrade/12.2_to_12.3_optional/mariadb.sql

```diff
@@ -0,0 +1,27 @@
+CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id ON oc_job_argument (id);
+CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id_argument ON oc_job_argument (id, argument);
```

Comment:
MariaDB [opencast]> CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id_argument ON oc_job_argument (id, argument);
ERROR 1071 (42000): Specified key was too long; max key length is 3072 bytes

Reply:
With Postgres it's the same:
opencast=# CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id_argument ON oc_job_argument (id, argument);
ERROR: Index row requires 19768 bytes maximum size is 8191